graviti/manager/sheets.py

```diff
@@ -63,7 +63,7 @@ def _init_data(self) -> None:
             )
             paging_lists = factory.create_lists(schema.get_keys())
             self._data[sheet_name] = DataFrame._from_paging(  # pylint: disable=protected-access
-                paging_lists, schema
+                paging_lists, schema  # type: ignore[arg-type]
```

Comment:
where these `ignore`s come from?

Reply:
The type of `schema` input of `Dataframe._from_paging` has been changed to record-like portex type, which is inconsistent with the return type of `PortexType.from_yaml`.